js/src/wp-seo-replacevar-plugin.js

```diff
@@ -529,5 +529,14 @@ var ReplaceVar = require( "./values/replaceVar" );
 		return parentText;
 	};
 
+	/**
+	 * STATIC VARIABLES
+	 */
+
+	/**
```

Comment:
As you've converted them to single-line comments, they should be just formatted as `// ... `

Though the initial intent was to separate blocks, thus having a block that stands out more is preferred.

The only thing that had to be adjust was converting from:
```
/**
 * SPECIALIZED REPLACES		
 */
```
to
```
/*
 * SPECIALIZED REPLACES		
 */
```